docs/guidelines/platforms/android/guideline_percievable_android.md

```diff
@@ -208,6 +208,181 @@ The content provided in your app must be easily understandable to all users. Tha
 
 ---
 
+
+### Resize text
+
+*This guideline covers point 1.4.4 Resize text - Level AA of the WCAG standard.*
+
+:white_check_mark: **Success criteria**
+
+Mobile apps must ensure that text can be resized up to **200%** without loss of content or functionality. This is essential for users with low vision who need larger text for readability. Specifically:
+
+- Text should be resizable up to **200%** using the device’s text size settings.
+- There should be no horizontal scrolling, content overlap, or loss of functionality when text is resized.
+- All UI components should remain usable and readable, even at larger text sizes.
+
+:no_entry_sign: **Failure criteria**
+
+- An app where text overflows or overlaps when scaled up to 200%, making it unreadable, would fail this criterion.
+- If an app forces horizontal scrolling to read resized text, it would also fail.
+- Defining text sizes in pixels (px) rather than sp would prevent proper scaling and lead to accessibility issues.
+
+**Bad Example:** Fixed Height with Text Encapsulation and limiting the line count
+
+```
+<TextView
+    android:id="@+id/fixed_text"
+    android:layout_width="wrap_content"
+    android:layout_height="50dp" <!-- Fixed height -->
+    android:maxLines="2"  <!-- Limits the text to 2 lines -->
+    android:ellipsize="end" <!-- Adds "..." at the end if text is too long -->
+    android:text="This is a long piece of text that might be cut off when resized."
+    android:textSize="16sp" />
+```
+
+```kotlin
+Box(modifier = Modifier.height(50.dp)) {
+    Text(
+        text = "This is a long piece of text that might be cut off when resized.",
+        fontSize = 16.sp,
+        maxLines = 2,  // Limits the text to 2 lines
+        overflow = TextOverflow.Ellipsis  // Adds "..." at the end if text is too long
+    )
+}
+```
+
+**Solution:** 
+
+```
+<TextView
+    android:id="@+id/fixed_text"
+    android:layout_width="wrap_content"
+    android:minHeight="50dp"  <!-- Minimum height but flexible -->
```

Comment:
If we have minHeight,layout_height is not needed

Reply:
In View-system, `android:layout_height` is mandatory since it defines basic/default size of the view. Without specifying `layout_height`, the layout system cannot determine the appropriate height for the TextView because `minHeight` only sets a minimum limit and doesn't define its default size. So to properly measure and position the view, the layout system needs a particular `android:layout_height` defined.

In Compose, on the other hand, content is wrapped by default so it is not needed to define this explicitly every time.